File: Data Structures & Algorithms/insertionSort/submission-0.cs
// Definition for a pair
// public class Pair {
//     public int Key;
//     public string Value;
//
//     public Pair(int key, string value) {
//         Key = key;
//         Value = value;
//     }
// }
public class Solution {
    public List<List<Pair>> InsertionSort(List<Pair> pairs) {
        List<List<Pair>> pairLog = new List<List<Pair>>();
        for(int i = 0; i < pairs.Count; i++) {
            int j = (i - 1);
            while(j >= 0 && (pairs[j + 1].Key < pairs[j].Key)) {
                Pair temp = new Pair(pairs[j + 1].Key, pairs[j + 1].Value);
                pairs[j + 1] = pairs[j];
                pairs[j] = temp;
                j--;
            }
            pairLog.Add(pairs.Select(p => new Pair(p.Key, p.Value)).ToList());
        }
        return pairLog;
    }
}

File: Data Structures & Algorithms/climbing-stairs/submission-0.js
class Solution {
    /**
     * @param {number} n
     * @return {number}
     */
    count = 0;
    climbStairs(n) {
        if(n <= 1) {
            return 1;
        }
        let val = this.climbStairs(n - 1) + this.climbStairs(n - 2);
        return val;
    }
}
